Assets/MixedRealityToolkit/_Core/Sharing/AnchorControl/AnchorSynchronizer.cs

```diff
@@ -0,0 +1,236 @@
+﻿using Pixie.AppSystems;
+using Pixie.Core;
+using Pixie.DeviceControl;
+using Pixie.StateControl;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Pixie.AnchorControl
+{
+    public class AnchorSynchronizer : MonoBehaviourSharingApp, ISharingAppObject, IAnchorSynchronizerClient, IAnchorSynchronizerServer
+    {
+        public const bool ApplyAlignmentRotation = true;
+        public const bool ApplyAlignmentScale = false;
+        public const bool RotateOnYAxisOnly = true;
+
+        public AnchorSyncStateEnum State { get { return state; } }
+        
+        [SerializeField]
+        private AnchorSyncStateEnum state = AnchorSyncStateEnum.Stopped;
+        private Dictionary<short, Transform> sharedAnchorTransformLookup = new Dictionary<short, Transform>();
+        private Dictionary<short, Transform> userAnchorTransformLookup = new Dictionary<short, Transform>();
+        private List<Transform> sharedAnchorTransformList = new List<Transform>();
+        private List<Transform> userAnchorTransformList = new List<Transform>();
+        private Transform sceneRootTransform;
```

Comment:
Is there an observable field in this anchor synchronizer that is intended to be a transform to the shared experience's origin?

Reply:
No, the transforms I'm using here are strictly helpers. The system exposes alignment info via state data exclusively. You could apply that data to a scene origin transform pretty easily.

The UserAlignment script applies the data to a transform holding the user's camera. That setup assumes you have a static scene (eg lightmapped) and want to align the users to the scene, not the other way around.

Reply:
Are there any known scenarios where we would have a non-static scene? Is there any thought on how content anchored to other users could be updated with this system?